docs/contracts/inheritance.rst

```diff
@@ -477,6 +490,10 @@ The reason for this is that ``C`` requests ``X`` to override ``A``
 requests to override ``X``, which is a contradiction that
 cannot be resolved.
 
+Due to the fact that you have to explicitly override a function
+that is inherited from multiple bases without a unique override,
+C3 linearization is not too important in practice.
```

Comment:
Can `C3 linearization` be briefly explained?

Reply:
C3 is introduced in this section which contains a link to a wikipedia article about it. It cannot be briefly explained, but its properties are briefly explained above.